e2e/pages/caseView.page.js

```diff
@@ -71,12 +71,13 @@ module.exports = {
   },
 
   checkTaskStatus(task, status) {
-    I.waitForElement(locate(`//p/a[text()="${task}"]`), 10);
+    const taskElement = `//p/a[text()="${task}"]`;
```

Comment:
I know that you didn't write this but if you wanted to have codecept build the locator for you you could write this as
```js
locate('p').withChild(locate('a').withText(task));
```
see https://codecept.io/locators/ for more on how that works

Reply:
yeah that's a fair point - is there a team preference on that format? personally I prefer using the raw xpath just because I feel I've got more control and less magic happening, but I'd noticed the Codecept locators and happy to follow that pattern if it's preferred?

Reply:
it looks to be about split in the e2e code, so i'll merge it in and if we want to go with one or other pattern i'm happy to update in a separate PR